target_chains/cosmwasm/tools/src/wormhole-stub.ts

```diff
@@ -0,0 +1,173 @@
+import { ChainId, createExecutorForChain } from "./chains-manager/chains";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { readFileSync } from "fs";
+import {
+  InstantiateContractResponse,
+  StoreCodeResponse,
+} from "./chains-manager/chain-executor";
+import { Pipeline, Step } from "./pipeline";
+import { CHAINS } from "@pythnetwork/xc-governance-sdk";
+const argv = yargs(hideBin(process.argv))
+  .usage("USAGE: npm run wormhole-stub -- <command>")
+  .option("mnemonic", {
+    type: "string",
+  })
+  .option("mainnet", {
+    type: "boolean",
+  })
+  .help()
+  .alias("help", "h")
+  .wrap(yargs.terminalWidth())
+  .parseSync();
+
+async function run() {
+  if (argv.mnemonic === undefined) {
+    console.log("Please provide the mnemonic");
+    return;
+  }
+
+  // IMPORTANT: IN ORDER TO RUN THIS SCRIPT FOR OTHER CHAINS
+  // WE NEED SOME METADATA
+  // HERE IS WHERE WE WILL ADDING THAT
+  // REPLACE THIS PART OF THE CODE FOR NEW CHAINS
+  const chainIds = [ChainId.NEUTRON_TESTNET_PION_1];
```

Comment:
We can automate this part too. Thinking about it though. When a new chain will be added we will have to add that chain metadata. This can be done with that step.